### DISCUSSION TITLE
TikTok Feature: change the content region

### DISCUSSION THREAD

**Discussion by Mattis142** [ReVanced/revanced-discussions#718] [Question]
Please add a patch to change the region of the app so people can access f.e regional features (like the asian beauty filters) and get content from selected countries, there's a different TikTok mod with this feature as well so I assume it's not too hard to replicate

**Comment by LisoUseInAIKyrios** [comment]
There now is location/sim spoof function

**Reply by EslamSad3** [reply]
how to access it

**Reply by LisoUseInAIKyrios** [reply]
Sim spoof is in the TikTok in app settings (under ReVanced menu)

**Reply by EslamSad3** [reply]
> Sim spoof is in the TikTok in app settings (under ReVanced menu)

i can't see the revanced menu
